database/factories/GameFactory.php

```diff
@@ -6,7 +6,7 @@
 
 use Illuminate\Database\Eloquent\Factories\Factory;
 use Tipoff\Scheduling\Models\Game;
-use Tipoff\Scheduling\Models\Slot;
+use Tipoff\Bookings\Models\Slot;
```

Comment:
@drewroberts  for example here, we need to call the Slot model from `Bookings` 

Another approach it would be instead of

'slot_id'               => Slot::factory()->create(['start_at' => $startingDate]),

do

  /** @var Model $slotModel */
  $slotModel = app('slot');
'slot_id'               =>  $slotModel::factory()->create(['start_at' => $startingDate]),

I haven't seen this before in the code, so I'd rather to ask you first.

Reply:
This package will require the Bookings package, so that is fine.